kibana-reports/server/routes/lib/createReport.ts

```diff
@@ -114,13 +114,24 @@ export const createReport = async (
           }
         });
       }
+      // If header exists assuming that it needs forwarding
+      let AdditionalHeaders: Headers | undefined;
```

Comment:
please change the var name to start with lower case letter, to keep the same style